i18n/locale-en.json

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
There should probably be a backend/frontend test that ensures the keys of each i18n dict are the same?